qiita_db/metadata_template.py

```diff
@@ -2228,20 +2229,32 @@ def load_template_to_dataframe(fn, strip_whitespace=True):
     |             longitude |        float |
     +-----------------------+--------------+
     """
+    # Load in file lines
+    holdfile = None
+    if isinstance(fn, (str, unicode)):
+        # filepath passed
+        with open(fn) as f:
+            holdfile = f.readlines()
+    else:
+        # open file passed
+        holdfile = fn.readlines()
```

Comment:
This block (lines 2232-2240) is the exact functionality that skbio.util.io.open_file provides, is there a reason not to use it?